pmultiqc/modules/quantms/quantms.py

```
@@ -1131,26 +1140,52 @@ def cal_heat_map_score(self):
+            pep_table = pep_table.merge(
+                right=self.file_df[["Sample", "Run"]].drop_duplicates(),
+                left_on="stand_spectra_ref",
+                right_on="Run"
+            )
+
+            pep_intensity_by_sample = dict()
+            pep_table["Sample"] = pep_table["Sample"].astype(int)
+            for name, group in pep_table.groupby("Sample", sort=True):
+
+                if config.kwargs["remove_decoy"]:
+                    pep_intensity_by_sample[f"Sample {str(name)}"] = group[
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)